### PR TITLE
fix: Basic setup #2

### LINKED ISSUE #2: Basic setup
Create a class related to all input for `decide`. This should include all of the assignment inputs

### PR DESCRIPTION
**Pull request by hampfh**
Co-authored-by: Pauline Garelli <pauline.garelli@hotmail.fr>
Co-authored-by: Edward Norberg <norberg.edward@gmail.com>
Co-authored-by: Erik Wachtmeister <erik.wac@gmail.com>

### CODE REVIEW

**Review by paulinegarelli — CHANGES_REQUESTED** [review]
Need to add constructors for classes Parameters and InitialSettings so that final values are initialized there (can't compile right now).

**Review by paulinegarelli — APPROVED** [review]

File: decide/src/main/java/org/example/InitialSettings.java [hunk 1]
```diff
@@ -22,13 +31,36 @@ public class Parameters {
...
+            F_PTS = f_pts;
+            G_PTS = g_pts;
+            LENGTH2 = length2;
+            RADIUS2 = radius2;
+            AREA2 = area2;
+        }
     }
 
-    public class Point {
+    public static class Point {
```
Comment (hampfh) [inline decide/src/main/java/org/example/InitialSettings.java]: Why do we make `Point` static?

**Review by hampfh — COMMENTED** [review]
Changes looks good, nice work @paulinegarelli!

File: decide/src/main/java/org/example/InitialSettings.java (hunk 1 above)
Comment (paulinegarelli) [reply decide/src/main/java/org/example/InitialSettings.java]: Oh yes you're right, that's a mistake I'll change it.

**Review by Storsorken — APPROVED** [review]
I think it'll do. 

One small naming error as pointed out by @Hampfh, but it'll be corrected when merging with condition 0.

The Points class is quite barebones as it doesn't yet support equality checks between instances but as long as we keep that in mind it could suffice

**Pull request merged**